src/reference/asciidoc/file.adoc

```diff
@@ -411,6 +411,10 @@ Examples of such events are:
 
 This sequence of events might occur, for example, when a file is rotated.
 
+`[message=Idle timeout, file=/tmp/foo] [idle time=5438]`
```

Comment:
This something unrelated.
Would be better to tell:
> Starting with _version 5.0_, a `FileTailingIdleEvent` is emitted when there is no data in the file during `idleEventInterval`.

As a general note. And only after that a sample.

This addition should be mentioned in the `What's New`, too.